matchms/networking/networking_functions.py

```diff
@@ -0,0 +1,44 @@
+""" Helper functions to build and handle spectral networks
+"""
+from typing import Tuple
+from matchms import Scores
+
+
+def get_top_hits(scores: Scores, identifier: str = "spectrumid",
+                 top_n: int = 25, search_by: str = "queries") -> Tuple[dict, dict]:
+    """Get top_n highest scores (and indices) for every entry.
+
+    Args:
+    --------
+    scores
+        Matchms Scores object containing all Spec2Vec similarities.
```

Comment:
Can also so be non-Spec2Vec similiarities
```suggestion
        Matchms Scores object containing all similarities.
```

Reply:
Thanks good catch.